### PR TITLE
[FEATURE] Add support for circles, graying, and total count on clues

### PR DESCRIPTION
**Pull request by ifilonenko**

### CODE REVIEW

File: src/XGrid.js [hunk 1]
```diff
@@ -12,22 +12,29 @@ export default function Grid({grid, selected, currentWord, hideAnswers, onClick}
...
         if(isSelected) {
           classNames.push("Grid-item-selected")
         } else if (isBlockedSquare(val)) {
           classNames.push("Grid-item-blocked")
         } else if (currentWord && currentWord.coordinates.some(i => i[0] === row && i[1] === column)) {
           classNames.push("Grid-item-highlight")
         }
+        if (isCircledSquare(val)) {
+          spanClassNames.push("Circled")
+          valLength = 1 // TODO: Can we have Rebus with circle?
```
Comment (kelaban) [inline src/XGrid.js:26]: On NYT they do this on Wednesdays with a rebus, so yeah it would be valid

File: src/MenuActions.js [hunk 2]
```diff
@@ -58,7 +64,11 @@ export default forwardRef(function MenuActions(props, ref) {
     handleInsertBlockedSquare: {
       key: BLOCKED_SQUARE,
       action: () => onRebus(BLOCKED_SQUARE)
-    }
+    },
+    handleInsertCircledLetter: {
+      key: 'mod+o',
```
Comment (kelaban) [inline src/MenuActions.js:69]: This is already used by the "open puzzle" command.

Alternatively, for blocked squares the menu just shows `.` even thought it's technically not a hotkey to teach a user how to add a black square. Does it make sense to ditch the form and do the same thing for this?

File: src/App.js [hunk 3]
```diff
@@ -351,6 +351,7 @@ function App() {
       onUndo={() => dispatchGridStateUpdate({type: UNDO_ACTION})}
       onRedo={() => dispatchGridStateUpdate({type: REDO_ACTION})}
       onRebus={(letter) => keyPressHandler.handleLetter(letter)}
+      onCircle={(letter) => keyPressHandler.handleLetter("~"+letter)}
```
Comment (kelaban) [inline src/App.js:354]: use the `CIRCLED_VALUE ` constant here

File: src/XGrid.js (hunk 1 above)
Comment (ifilonenko) [reply src/XGrid.js:26]: bleh, then there will need to be some finagling here.... so would we just be supporting a "rebus" like form?

File: src/MenuActions.js (hunk 2 above)
Comment (ifilonenko) [reply src/MenuActions.js:69]: Oh ditching the form would probably be best, but what happens when it is empty?. I guess we would reject?

**Comment by ifilonenko** [comment]
I resolved comments, the new method of interacting with the cells is: 
upon doing: `cmd+c` you create a circle. When running `cmd+c` on an empty cell nothing happens, when doing `cmd+c` on an existing "circled" cell, it `un-circles`

**Pull request merged**